src/Test/Utilities/Portable/Assert/ConditionalFactAttribute.cs

```diff
@@ -157,6 +158,7 @@ public static class ExecutionConditionUtil
 
         public static bool IsWindows => Path.DirectorySeparatorChar == '\\';
         public static bool IsUnix => !IsWindows;
+        public static bool IsMacOS => RuntimeInformation.IsOSPlatform(OSPlatform.OSX); // NOTE: may not be valid on mono
```

Comment:
Not sure if we want to keep this comment or not. If there's an actual problem people need to be aware of let's link to a tracking issue.

Reply:
Lets delete the comment. There is no known active bug here.

Reply:
Done